tests/internal/test_deps.py

```diff
@@ -119,22 +128,34 @@ def test_pkg_info(self, get_distribution_mock):
         assert "setuptools" == dep_info_setuptools["name"]
         assert "0.6" == dep_info_setuptools["version"]
 
-    @mock.patch("pkg_resources.get_distribution")
+    @mock.patch("importlib_metadata.distribution")
     def test_pkg_info_for_missing_dist(self, get_distribution_mock):
-        get_distribution_mock.side_effect = pkg_resources.DistributionNotFound
+        get_distribution_mock.side_effect = metadata.PackageNotFoundError(
+            "test"
+        )
 
         result = deps.pkg_info()
 
         assert "Mopidy" == result["name"]
         assert "version" not in result
         assert "path" not in result
 
-    @mock.patch("pkg_resources.get_distribution")
+    @pytest.mark.skip("Version control missing in metadata")
+    @mock.patch("importlib_metadata.distribution")
     def test_pkg_info_for_wrong_dist_version(self, get_distribution_mock):
-        get_distribution_mock.side_effect = pkg_resources.VersionConflict
-
+        # get_distribution_mock.side_effect = metadata.VersionConflict
         result = deps.pkg_info()
 
         assert "Mopidy" == result["name"]
         assert "version" not in result
         assert "path" not in result
+
+    @pytest.mark.parametrize("include_transitive_deps", [True, False])
+    @pytest.mark.parametrize("include_extras", [True, False])
+    def test_pkg_info_real(self, include_transitive_deps, include_extras):
+        result = deps.pkg_info(
+            "Mopidy",
+            include_transitive_deps=include_transitive_deps,
+            include_extras=include_extras,
+        )
+        assert result
```

Comment:
OK, I see, this is assert only because it's unpredictable real data rather than mocked?

Reply:
yup.